types/rbac_test.go

```diff
@@ -1,6 +1,7 @@
 package types
 
 import (
+	"sensu-go/types"
```

Comment:
Just "types" here.

Reply:
@bitmonk this file is already in the `types` package, so there shouldn't be an import of `types` here, and you don't need to refer to anything with the `types`package qualifier.

Try running `go build` in the package. If the command succeeds, your imports are correct. You can also automatically manage your imports with goimports. (`go get golang.org/x/tools/cmd/goimports`)

Reply:
@echlebek thanks!